docs/tutorials/access-service-across-clusters.md

```diff
@@ -19,7 +19,7 @@ Ensure that at least two clusters have been added to Karmada, and the container
 * If you use the `hack/local-up-karmada.sh` script to deploy Karmada, Karmada will have three member clusters, and the container networks of the member1 and member2 will be connected.
 * You can use `Submariner` or other related open source projects to connected networks between member clusters.
 
-Note: In order to prevent routing conflicts, Pod and Service CIDRs of clusters need non-overlapping.
+> Note: In order to prevent routing conflicts, Pod and Service CIDRs of clusters need non-overlapping.
```

Comment:
```suggestion
:::note
In order to prevent routing conflicts, Pod and Service CIDRs of clusters need non-overlapping.
:::
```
It's recommended to try [the note style specific to Docusaurus](https://docusaurus.io/docs/2.x/markdown-features/admonitions).

Reply:
Done. Thanks!